assets/scss/theme.scss

```diff
@@ -68,10 +69,10 @@
 //
 // -----------------------------------------------------------------------------
 
+@import "common/index"; // 6
```

Comment:
Could we place it by order?

Reply:
Common/index has tooltip. If it will be placed at the bottom, css files using it will not see tooltip statement. But I think we can place it before @import "components/components”; - on penultimate place

Reply:
Up to you